pcsx2/GS/Renderers/Common/GSVertexTrace.cpp

```diff
@@ -153,6 +151,20 @@ void GSVertexTrace::Update(const void* vertex, const u32* index, int v_count, in
 	}
 }
 
+template <GS_PRIM_CLASS primclass, u32 iip, u32 tme, u32 fst, u32 color>
+GSVertexTrace::FindMinMaxPtr GSVertexTrace::GetFMM(bool provoking_vertex_first)
+{
+	constexpr bool real_iip = primclass == GS_SPRITE_CLASS ? false : iip;
+	constexpr bool real_fst = tme ? fst : false;
+	bool provoking_vertex_first_class = primclass == GS_LINE_CLASS || primclass == GS_TRIANGLE_CLASS;
+	bool swap = provoking_vertex_first_class && !iip && provoking_vertex_first;
```

Comment:
These two can be constants, didn't check the vectors if they are set again so maybe those as well.